src/main/java/org/cloudfoundry/promregator/scanner/ResolvedTarget.java

```diff
@@ -14,6 +14,8 @@
 	private String path;
 	
 	private String protocol;
+
+	private String id;
```

Comment:
I just got here some second thoughts: `id` is very generic --- and it would not be the id of a `ResolvedTarget`. 
Can we agree on renaming it to `applicationId`?

Reply:
Happy to rename it. I'll do that first think when I get into work.

Reply:
This change is done now.